### PR TITLE
minor fix in pump client

### PR DESCRIPTION
**Pull request by WangXiangUSTC**

### CODE REVIEW

File: tidb-binlog/pump_client/selector.go [hunk 1]
```diff
@@ -218,8 +218,7 @@ func (r *RangeSelector) Next(pump *PumpStatus, binlog *pb.Binlog, retryTime int)
 
 	nextPump := r.Pumps[r.Offset]
 	if binlog.Tp == pb.BinlogType_Prewrite {
-		r.Offset++
```
Comment (WangXiangUSTC) [inline tidb-binlog/pump_client/selector.go:221]: already add offset in L206

**Comment by july2993** [comment]
LGTM

**Comment by IANTHEREAL** [comment]
LGTM

File: tidb-binlog/pump_client/selector.go [hunk 2]
```diff
@@ -116,7 +116,7 @@ func (h *HashSelector) Next(pump *PumpStatus, binlog *pb.Binlog, retryTime int)
 
 	nextPump := h.Pumps[(hashTs(binlog.StartTs)+int(retryTime))%len(h.Pumps)]
 	if binlog.Tp == pb.BinlogType_Prewrite {
-		h.TsMap[binlog.StartTs] = pump
+		h.TsMap[binlog.StartTs] = nextPump
```
Comment (IANTHEREAL) [inline tidb-binlog/pump_client/selector.go:119]: what's any useful of pump parameter?

Comment (WangXiangUSTC) [reply tidb-binlog/pump_client/selector.go:119]: useless, I will remove it

**Comment by WangXiangUSTC** [comment]
PTAL again @GregoryIan

File: tidb-binlog/pump_client/client_test.go [hunk 3]
```diff
@@ -110,4 +110,30 @@ func (*testClientSuite) testPumpsClient(c *C, algorithm string) {
...
+		}
+
+		pump1 := pumpsClient.Selector.Select(prewriteBinlog)
+		if j%2 == 0 {
+			pump1 = pumpsClient.Selector.Next(prewriteBinlog, 0)
+		}
+
+		pumpsClient.setPumpAvaliable(pump1, false)
+		pump2 := pumpsClient.Selector.Select(commitBinlog)
+		c.Assert(pump2.IsAvaliable, Equals, false)
```
Comment (IANTHEREAL) [inline tidb-binlog/pump_client/client_test.go:131]: add `c.Assert(pump1.NodeID, Equals, pump2.NodeID)`

Comment (WangXiangUSTC) [reply tidb-binlog/pump_client/client_test.go:131]: done

**Comment by IANTHEREAL** [comment]
LGTM

**Pull request merged**